pdns/recursordist/docs/changelog/4.5.rst

```diff
@@ -1,5 +1,54 @@
 Changelogs for 4.5.X
 ====================
+.. changelog::
+  :version: 4.5.0-beta1
+  :released: 26th of March 2021
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9995
+    :tickets: 7982
+
+    Support tcp fastopen connect.
```

Comment:
Perhaps we could make it more clear than it is for outgoing connections, something like:
```suggestion
    Support tcp fastopen connect on outgoing connections.
```